### PR TITLE
Instrument crate with tracing instead of log crate

### PR DESCRIPTION
**Pull request by slawlor**
Once instrumentation is included on the actor handlers:

```text
$ cargo run --example counter
   Compiling ractor v0.8.5 (/Users/seanlawlor/src/slawlor_ractor/ractor)
    Finished dev [unoptimized + debuginfo] target(s) in 2.23s
     Running `target/debug/examples/counter`
I0805 21:23:53.769058 93424 ractor/examples/counter.rs:46] [Actor{id: "0.0"}] Starting the counter actor
I0805 21:23:53.770157 93424 ractor/examples/counter.rs:57] [Actor{id: "0.0"}] Counting actor handle message...
I0805 21:23:53.770190 93424 ractor/examples/counter.rs:57] [Actor{id: "0.0"}] Counting actor handle message...
I0805 21:23:53.770211 93424 ractor/examples/counter.rs:57] [Actor{id: "0.0"}] Counting actor handle message...
I0805 21:23:53.770228 93424 ractor/examples/counter.rs:57] [Actor{id: "0.0"}] Counting actor handle message...
I0805 21:23:53.770262 93424 ractor/examples/counter.rs:124] Count is: 10
I0805 21:23:53.770322 93424 ractor/examples/counter.rs:57] [Actor{id: "0.0"}] Counting actor handle message...
I0805 21:23:53.770342 93424 ractor/examples/counter.rs:57] [Actor{id: "0.0"}] Counting actor handle message...
I0805 21:23:53.770359 93424 ractor/examples/counter.rs:57] [Actor{id: "0.0"}] Counting actor handle message...
I0805 21:23:53.770375 93424 ractor/examples/counter.rs:57] [Actor{id: "0.0"}] Counting actor handle message...
I0805 21:23:53.770407 93424 ractor/examples/counter.rs:124] Count is: 20
I0805 21:23:53.770459 93424 ractor/examples/counter.rs:57] [Actor{id: "0.0"}] Counting actor handle message...
I0805 21:23:53.770494 93424 ractor/examples/counter.rs:57] [Actor{id: "0.0"}] Counting actor handle message...
I0805 21:23:53.770515 93424 ractor/examples/counter.rs:57] [Actor{id: "0.0"}] Counting actor handle message...
I0805 21:23:53.770534 93424 ractor/examples/counter.rs:57] [Actor{id: "0.0"}] Counting actor handle message...
I0805 21:23:53.770569 93424 ractor/examples/counter.rs:124] Count is: 30
I0805 21:23:53.770626 93424 ractor/examples/counter.rs:57] [Actor{id: "0.0"}] Counting actor handle message...
I0805 21:23:53.770652 93424 ractor/examples/counter.rs:57] [Actor{id: "0.0"}] Counting actor handle message...
I0805 21:23:53.770673 93424 ractor/examples/counter.rs:57] [Actor{id: "0.0"}] Counting actor handle message...
I0805 21:23:53.770693 93424 ractor/examples/counter.rs:57] [Actor{id: "0.0"}] Counting actor handle message...
I0805 21:23:53.770727 93424 ractor/examples/counter.rs:124] Count is: 40
```

### CODE REVIEW

File: ractor_cluster/src/net/listener.rs [hunk 1]
```diff
@@ -110,7 +110,10 @@ impl Actor for Listener {
                                     stream: enc_stream,
                                 }),
                                 Err(some_err) => {
-                                    log::warn!("Error establishing secure socket: {}", some_err);
+                                    tracing::warn!(
+                                        "Error establishing secure socket: {}",
+                                        some_err
+                                    );
```
Comment (jsgf) [inline ractor_cluster/src/net/listener.rs:116]: Nit: use the `{some_err}` syntax? Or is there an MSRV constraint? Or just don't like it?

**Review by jsgf — APPROVED** [review]
Unless I missed something, is this just a mechanical `s/log::/tracing::/g`?

Once tracing is in place there'll be a lot more we'll want to record, but this is a good start.

File: ractor_cluster/src/net/listener.rs (hunk 1 above)
Comment (slawlor) [reply ractor_cluster/src/net/listener.rs:116]: Hmm I thought clippy caught these, this is just a general oversight. Will fix.

Comment (slawlor) [reply ractor_cluster/src/net/listener.rs:116]: Hmm I thought clippy caught these, this is just a general oversight. Ah perhaps my local clippy is out of date, will re-run or fix manually. thanks

**Comment by slawlor** [comment]
> Unless I missed something, is this just a mechanical `s/log::/tracing::/g`?
> 
> Once tracing is in place there'll be a lot more we'll want to record, but this is a good start.

It is in addition to instrumenting 2 calls in the actor handler with the actor's id. This adds a span so that tracing messages called from an actor handler will attach the actor id which means we can get sequential message flow from actors grouped by actor id. Agree it's a start, and there's room to grow.

File: ractor_cluster/src/net/listener.rs (hunk 1 above)
Comment (slawlor) [reply ractor_cluster/src/net/listener.rs]: https://github.com/slawlor/ractor/pull/134/commits/fa9faa2764262acc421c5b7ee4626acffd50e98e#diff-404312f7298c59a6ebaf75fdf76f00c44ecb4351fbe48e9d07d8bd71f1d57913

**Comment by slawlor** [comment]
Related issue #22

**Pull request merged**